module/VuFind/src/VuFind/Search/EDS/Params.php

```diff
@@ -47,6 +47,16 @@
  */
 class Params extends AbstractEDSParams
 {
+    /**
+     * Fields that the EDS API will always filter multiple values using OR, not AND.
+     *
+     * @var array
+     */
+    protected $FORCED_OR_FIELDS = [
```

Comment:
There is [conversation](https://vufind.slack.com/archives/C04GRUNA2/p1715003317040519) about this in the VuFind slack from May 10th, with @cwolfebsco and @berndoberknapp.  I'm duplicating some of it here since that slack thread will expire in about a month.

Claus Wolf
  2 months ago
SourceTypes should be mutually exclusive, i.e. a book cannot be an article, so would we truly expect an AND to be possible here? The only SourceType I can think that are no mutually exclusive are books and ebooks


Bernd Oberknapp
  [2 months ago](https://vufind.slack.com/archives/C04GRUNA2/p1715574546273229?thread_ts=1715003317.040519&cid=C04GRUNA2)
No, but this is also the case for other facets like ContentProvider and Journal. I would expect all facets to behave in the same way, and if the client requests AND the result should be empty.


Claus Wolf
  2 months ago
I'll be happy to pass this feedback to our development teams. ContentProvider & SourceType have always been somewhat special, even in the native UI. I'll write it up and provide you with an Enhancement Request Number.


Maccabee Levine
  [2 months ago](https://vufind.slack.com/archives/C04GRUNA2/p1715588120676529?thread_ts=1715003317.040519&cid=C04GRUNA2)
Thank you Claus for the help.  I agree with Bernd that the behavior is unexpected (and worse, I think it's undocumented)


Claus Wolf
  2 months ago
This has been recorded as Enhancement Request ER 3366702. I did copy the Product Manager for the API for his awareness